solaris/solaris11/postinstall.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# postinst script for wazuh-agent
+# Wazuh, Inc 2015-2020
+
+if [ -d <INSTALL_PATH>/logs/ossec ]; then
+  mv <INSTALL_PATH>/logs/ossec/* <INSTALL_PATH>/logs/wazuh
```

Comment:
Moving log files without asking the user? very sensitive change.

Reply:
After this issue wazuh/wazuh#6885, `var/ossec/logs/ossec` will ont exist anymore this allows us to maintain the old logs.